Tests/test_Probcons_tool.py

```diff
@@ -77,9 +82,11 @@ def test_Probcons_complex_commandline(self):
         cmdline.set_parameter("--iterative-refinement", 222)
         cmdline.set_parameter("a", True)
         cmdline.annot = self.annotation_outfile
-        self.assertEqual(str(cmdline), probcons_exe +
-                         " -c 4 -ir 222 -pre 1 -annot Fasta/probcons_annot.out"
-                         " -a Fasta/fa01")
+        self.assertEqual(
+            str(cmdline),
+            probcons_exe + " -c 4 -ir 222 -pre 1 -annot Fasta/probcons_annot.out"
+            " -a Fasta/fa01",
+        )
```

Comment:
Does this look better with the string literal in one (currently split in two with implicit concatenation)?

Reply:
Yes, I think the literal can go on one line, even if black splits the addition on 2 lines, reads better. Thank you.